cegs_portal/search/views/v1/experiment.py

```diff
@@ -31,12 +32,42 @@ def is_archived(self):
         except ObjectNotFoundError as e:
             raise Http404(str(e))
 
+    def request_options(self, request):
+        """
+        GET queries used:
+            analysis (single)
+                * Should be a valid analysis accession ID
+        """
+        options = super().request_options(request)
+        options["analysis"] = request.GET.get("analysis", None)
+        if options["analysis"] is not None:
+            validate_analysis_accession_id(options["analysis"])
+
+        return options
+
     def get(self, request, options, data, exp_id):
+        analyses = list(ExperimentSearch.all_analysis_id_search(exp_id))
+        analyses_list = []
+
+        if options["analysis"] is not None:
+            analysis_selected = options["analysis"]
+        else:
+            analysis_selected = ExperimentSearch.default_analysis_id_search(exp_id)
+
+        for analysis in analyses:
+            if analysis == options["analysis"]:
+                analyses_list.append(("selected", analysis))
+            else:
+                analyses_list.append(("", analysis))
+
         return super().get(
             request,
             options,
             {
                 "logged_in": not request.user.is_anonymous,
+                "analyses": analyses,
+                "analyses_list": analyses_list,
+                "analysis_selected": analysis_selected,
```

Comment:
What happens if `analysis_selected` is `None`? (as might be the case when an experiment doesn't have an analysis uploaded yet)

Reply:
Would they be able to submit without an analysis? Would not submission errors show up? If no, what should we set it to?

Reply:
Yes, you can have an experiment without an analysis. It's kind of a corner case, but it's possible. 

I don't think there should be a "v1" beside the experiment and I think the overview/coverage tab should show a reasonable error message. I think the analysis graph tab already will show an acceptable error message.

Reply:
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/7c114f5a-8fad-48a2-bb98-1b06a0982db6)

Reply:
Oh good, I think that's fine.

Reply:
Everyone in the meeting wanted the version dropdown the other week. Maybe there can be a conversation about it Friday so that I dont have to keep editing it on this PR.

Reply:
If there is no analysis there should be no version dropdown. Nor does it make sense to have "v1".

Reply:
Got it. Right, that makes sense.